spring-cloud-alibaba-tests/sentinel-tests/sentinel-flowcontrol-test/src/test/java/com/alibaba/cloud/tests/sentinel/degrade/SentinelFlowControlTestAppTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2012-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.tests.sentinel.degrade;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+class SentinelFlowControlTestAppTest {
+
+	@LocalServerPort
+	int port;
+
+	@Test
+	public void testFlowControl() throws InterruptedException {
+		RestTemplate rest = new RestTemplate();
```

Comment:
Maybe I think it's better that set threadCount is 2, and check if there is fallback signal in result set first. Secondly, call a method again, and check is there is fallback signal in result set. Only in this way, we can conclude the flow control threshold of 2 is valid.